### PR TITLE
12.0 mig pos fix search limit wp

### PR DESCRIPTION
**Pull request by wpichler**
Just bumbed version number to 12 - is a dependency for #411

### CODE REVIEW

**Comment by legalsylvain** [comment]
WP means Work in Progress ? 
is this PR ready for review ? 

regards.

**Comment by wpichler** [comment]
WP means Wolfgang Pichler ;-)
The PR is ready for review.
I will not use -WP for my next PR's - thanks for the hint

**Comment by legalsylvain** [comment]
Hi,
Thanks for this PR.
1) could you split readme file, into many rst file ? (New OCA convention)
https://github.com/OCA/maintainer-tools/tree/master/template/module/readme
Oca bot will generate correct readme file.

2) don't know how to test, as there are a very few quantity of product in odoo demo products.

Do you have an idea ?

Cc : @hparfr

File: pos_fix_search_limit/static/src/js/pos_fix_search_limit.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            if(this.el && this.el.parentNode){
+                this.el.parentNode.replaceChild(el_node,this.el);
+            }
+            this.el = el_node;
+
+            var list_container = el_node.querySelector('.product-list');
+            for(i=0; i < len; i++){
+                product_node = this.render_product(this.product_list[i]);
+                product_node.addEventListener('click',this.click_product_handler);
+                frag.appendChild(product_node);
```
Comment (hparfr) [inline pos_fix_search_limit/static/src/js/pos_fix_search_limit.js:45]: Add : 
```
             product_node.addEventListener('keypress',this.keypress_product_handler); 

```
It's the only usefull diff I see between v10 and v12 in this file.

**Review by hparfr — CHANGES_REQUESTED** [review]
Thanks, just a minor addition to be sync with the v12 version of pos.

@legalsylvain  I guess it can be tested manually.
The code didn't change much from v10 to v12 so I guess this code is fine.

File: pos_fix_search_limit/static/src/js/pos_fix_search_limit.js (hunk 1 above)
Comment (hparfr) [reply pos_fix_search_limit/static/src/js/pos_fix_search_limit.js]: https://github.com/odoo/odoo/blame/864357203fc6222cc989b99a5e9f428cd53f413f/addons/point_of_sale/static/src/js/screens.js#L1927

Comment (hparfr) [reply pos_fix_search_limit/static/src/js/pos_fix_search_limit.js:45]: See https://github.com/odoo/odoo/blob/12.0/addons/point_of_sale/static/src/js/screens.js#L984

Comment (wpichler) [reply pos_fix_search_limit/static/src/js/pos_fix_search_limit.js:45]: You are correct - i've added the keypress eventhandler - thanks

**Review by hparfr — APPROVED** [review]
Thanks

**Comment by wpichler** [comment]
Please merge this PR - the pos_product_template PR does depend on this one...

**Pull request merged**